{{cookiecutter.project_slug}}/.github/workflows/ci.yml

```diff
@@ -98,6 +101,12 @@ jobs:
           python -m pip install --upgrade pip
           pip install -r requirements/local.txt
 
+      - name: Check DB Migrations
+        run: python manage.py makemigrations --check
+
+      - name: Run DB Migrations
+        run: python manage.py migrate
```

Comment:
I added this so the non-docker github ci is consistent with the docker github ci but happy to remove it if you think it's unnecessary or out of scope.